README.md

```diff
@@ -1,46 +1,296 @@
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+### Get an overview
 
-### `npm test`
+First, metadata and images for NFT must be stored in storage.
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+For example, if you wish to issue 30 NFTs, create directories and files like the following.
 
-### `npm run build`
+- images
+  - 0.png
+  - 1.png
+  - 2.png
+  - ...
+  - 29.png
+- json
+  - 0.json
+  - 1.json
+  - 2.json
+  - ...
+  - 29.json
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+The JSON file looks like this. 
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+Since the image field requires the URI of the image to be entered, the first step is to save the images somewhere and then enter the URI in the JSON.
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+`0.json`:
+```json
+{
+  "name": "Bunzz #0",
+  "description": "Bunzz NFT",
+  "image": "<URI for image>",
+  "edition": 0,
+  "date": 1664941479203,
+  "attributes": [
+    {
+      "trait_type": "Background",
+      "value": "Black"
+    },
+    {
+      "trait_type": "Bottom lid",
+      "value": "High"
+    },
+    {
+      "trait_type": "Top lid",
+      "value": "Middle"
+    }
+  ],
+}
+```
 
-### `npm run eject`
+### Download hashlips
 
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+These JSON and images can be created manually, but there is a useful tool called hashlips that can be used to create generative NFT.
 
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+https://github.com/HashLips/hashlips_art_engine
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
+We will use this repository, so download it.
+```bash
+$ git clone https://github.com/HashLips/hashlips_art_engine
+```
 
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
+Go to the hashlips directory and install the dependencies. 
 
-## Learn More
+**Note:** I used node version `14.18.x`, because hashlips had an error when using node version `16.x`.
```

Comment:
Hmm, should we put a sentence like this or kinda like `Use node version 14.18.x ...` or keep `I used node version`🤔

What do you think about it?

Reply:
I agree! Thank you